src/librustc/hir/mod.rs

```diff
@@ -22,7 +22,7 @@ pub use self::Mutability::*;
 pub use self::PrimTy::*;
 pub use self::Stmt_::*;
 pub use self::Ty_::*;
-pub use self::TyParamBound::*;
+pub use self::ParamBound::*;
```

Comment:
If you remove this, I think we can have `ParamBound::Trait` and `ParamBound::Outlives`.